Source/ASTableView.mm

```diff
@@ -1366,65 +1363,11 @@ - (ASRangeController *)rangeController
     return _rangeController;
 }
 
-- (NSArray *)visibleNodeIndexPathsForRangeController:(ASRangeController *)rangeController
```

Comment:
This code was also added for similar reasons.  At one point @eanagel had a reproducible test case for the scenario this addressed.

Could we split these changes out into a diff that is separate from the optimization, to be able to better understand the cause of any ASTableView-specific regressions?  I'm thinking we should be cautious here mainly because the largest clients are not using ASTableView very much, especially in dynamic / editing environments where this issue was occurring.

These changes were able to be avoided in https://github.com/facebook/AsyncDisplayKit/pull/3102, so I think it will work from a technical perspective to separate them.